apps/librelingo_yaml_loader/librelingo_yaml_loader/yaml_loader.py

```diff
@@ -403,6 +404,7 @@ def load_course(path):
         modules=[],
         settings=None,
         repository_url=course["Repository"],
+        course_dir=str(path),
```

Comment:
why `str(path)` instead of just `path`?

Reply:
https://github.com/kantord/LibreLingo/pull/1884/files#r774698354

Reply:
ah ok I think I get it. it's a Path type instead of string.

I'm not sure which one is better to use. string is much simpler which is great, on the other hand Path is built in in python as well. so it could also be useful to provide that type here as well and rather convert it when necessary to display and error message or such